Tasks/KubernetesManifestV0/src/actions/deploy.ts

```diff
@@ -6,6 +6,8 @@ import * as utils from '../utils/utilities';
 import { Kubectl } from 'kubernetes-common-v2/kubectl-object-model';
 
 export async function deploy(ignoreSslErrors?: boolean) {
+    TaskInputParameters.validateCanaryPercentage();
+    TaskInputParameters.validateReplicaCount();
```

Comment:
You could skip this getting called altogether. By calling the logic in the file itself.

Reply:
If the call is made in the same file, the errors thrown by these validation functions will not be caught by the catch block in run.ts and will remain unhandled.